target/ppc/translate/fixedpoint-impl.c.inc

```diff
@@ -124,9 +138,9 @@ static bool do_ldst_quad(DisasContext *ctx, arg_D *a, bool store, bool prefixed)
         }
     } else {
         if (store) {
-            tcg_gen_qemu_st_i64(tgt_a, ea, ctx->mem_idx, mop);
+            tcg_gen_qemu_st_i64(low_addr_gpr, ea, ctx->mem_idx, mop);
         } else {
-            tcg_gen_qemu_ld_i64(tgt_a, ea, ctx->mem_idx, mop);
+            tcg_gen_qemu_ld_i64(low_addr_gpr, ea, ctx->mem_idx, mop);
         }
 
         tcg_gen_addi_i64(ea, ea, 8);
```

Comment:
Just noticed gen_addr_add(ctx, ea, ea, 8) does all this. Is there a reason not to use it?

Reply:
And I would really define ea as TCGv, to match all uses of it, regardless of the ``#if defined(TARGET_PPC64)``  :)

Reply:
`lq{,arx}` and `stq{,cx}` are the only callers of `gen_addr_add`, so I was hoping to get rid of it. I'll change `ea` to `TCGv`.